### PR TITLE
ciphersuite recommendations

### LINKED ISSUE #7: Applied AEAD limits in (D)TLS & CCM_8 fate
See https://mailarchive.ietf.org/arch/msg/cfrg/vgPiYAlu03G9_Mx1yx_IEjirZVE/ in particular regarding CCM_8

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fixes #7

### CODE REVIEW

File: draft-ietf-uta-tls13-iot-profile.md [hunk 1]
```diff
@@ -376,8 +384,38 @@ infrastructure, tool support).
...
+
+Since all the ciphersuites mandated by {{RFC7925}} and {{CoAP}} are based on
+CCM_8, there is no stand-by ciphersuite to use for applications that want to
+avoid the security and availability risks associated with CCM_8 while retaining
+interoperability with the rest of the ecosystem.
+
+In order to ameliorate the situation, this document RECOMMENDS that
+implementations support
+
+* TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
```
Comment (boaks) [inline draft-ietf-uta-tls13-iot-profile.md:410]: I would still prefer to see `TLS_ECDHE_ECDSA_WITH_AES_128_CCM`.

Maybe something as 

Servers are recommended to support:
`TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`
and `TLS_ECDHE_ECDSA_WITH_AES_128_CCM`

And device should support:
`TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`
or, if the first is not available,
`TLS_ECDHE_ECDSA_WITH_AES_128_CCM`

Comment (thomas-fossati) [reply draft-ietf-uta-tls13-iot-profile.md:410]: thanks for the feedback Achim.  I wanted to keep the extra requirements down to a minimum though I understand where you are coming from.  @hannestschofenig what do you think of Achim's suggestion?

Comment (hannestschofenig) [reply draft-ietf-uta-tls13-iot-profile.md:410]: The suggestion by Achim sounds reasonable to me.

Comment (thomas-fossati) [reply draft-ietf-uta-tls13-iot-profile.md:410]: ACK, done in 14fde4e

**Pull request merged**